app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,74 @@
+class TweetsController < ApplicationController
+	get '/tweets' do
+		if logged_in?
+			@tweets = Tweet.all
+			erb :'tweets/tweets'
+		else
+			redirect to '/login'
+		end
+	end
+
+	get '/tweets/new' do
+		if logged_in?
+			erb :'tweets/create_tweet'
+		else
+			redirect to '/login'
+		end
+	end
+
+	get '/tweets/:id/edit' do
+		if logged_in?
+			@tweet = Tweet.find_by_id(params[:id])
+			if @tweet.user_id == current_user.id
+				erb :'tweets/edit_tweet'
+			else
+				redirect to '/tweets'
+			end
+		else
+			redirect to '/login'
+		end
+	end
+
+	get '/tweets/:id' do
+		if logged_in?
+			@tweet = Tweet.find_by_id(params[:id])
+			erb :'tweets/show_tweet'
+		else
+			redirect to '/login'
+		end
+	end
+
+	post '/tweets' do
+		if params[:content] == ""
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html#presence) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.